tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/contact/ContactAddedRabbitMqMessage.java

```diff
@@ -2,13 +2,18 @@
 
 import java.io.IOException;
 
+import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
 public record ContactAddedRabbitMqMessage(String bookId, String bookName, String contactId,
                                           String userId, JCardObject vcard) {
 
     private static final ObjectMapper objectMapper = new ObjectMapper();
 
+    static {
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    }
```

Comment:
How about `@JsonIgnoreProperties(ignoreUnknown = true)`?

Reply:
I see `objectMapper` here is only for `ContactAddedRabbitMqMessage`. Then no problem.
I'm fine with both way :)

Reply:
But static blocks look ugly and a little bit hacky :) Annotations, on the other hand, fit the picture.

Reply:
Let's stay focussed!
First working version will get merged. 
If you care enough to then open a fixup PR it will be accepted.